package.json

```diff
@@ -18,9 +18,10 @@
     "lint:spellcheck": "spellchecker --quiet --files=\"docs/**/*.md\" --dictionaries=\"./.spellcheck.dict.txt\" --reports=\"spelling.json\" --plugins spell indefinite-article repeated-words syntax-mentions syntax-urls frontmatter",
     "tsc:compile": "tsc --project .",
     "lint:all": "yarn lint && yarn lint:markdown && yarn lint:spellcheck && yarn tsc:compile",
-    "tests:jest": "jest",
-    "tests:jest-watch": "jest --watch",
-    "tests:jest-coverage": "jest --coverage",
+    "tests:vertex:mocks": "./vertex_mock_responses.sh && yarn ts-node ./packages/vertexai/__tests__/test-utils/convert-mocks.ts",
```

Comment:
maybe vertex_mock_responses.sh should be in `scripts` directory instead of root?